src/StackExchange.Redis/PhysicalBridge.cs

```diff
@@ -776,34 +841,40 @@ private bool PushToBacklog(Message message, bool onlyIfExists)
             StartBacklogProcessor();
             return true;
         }
+
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         private void StartBacklogProcessor()
         {
             if (Interlocked.CompareExchange(ref _backlogProcessorIsRunning, 1, 0) == 0)
             {
-
 #if DEBUG
                 _backlogProcessorRequestedTime = Environment.TickCount;
 #endif
                 _backlogStatus = BacklogStatus.Activating;
 
-                // start the backlog processor; this is a bit unorthadox, as you would *expect* this to just
+                // Start the backlog processor; this is a bit unorthadox, as you would *expect* this to just
                 // be Task.Run; that would work fine when healthy, but when we're falling on our face, it is
                 // easy to get into a thread-pool-starvation "spiral of death" if we rely on the thread-pool
                 // to unblock the thread-pool when there could be sync-over-async callers. Note that in reality,
                 // the initial "enough" of the back-log processor is typically sync, which means that the thread
                 // we start is actually useful, despite thinking "but that will just go async and back to the pool"
-                var thread = new Thread(s => ((PhysicalBridge)s).ProcessBacklogAsync().RedisFireAndForget());
-                thread.IsBackground = true; // don't keep process alive (also: act like the thread-pool used to)
-                thread.Name = "redisbacklog"; // help anyone looking at thread-dumps
+                var thread = new Thread(s => ((PhysicalBridge)s).ProcessBacklogsAsync().RedisFireAndForget())
+                {
+                    IsBackground = true,                  // don't keep process alive (also: act like the thread-pool used to)
+                    Name = "StackExchange.Redis Backlog", // help anyone looking at thread-dumps
+                };
                 thread.Start(this);
             }
         }
 #if DEBUG
         private volatile int _backlogProcessorRequestedTime;
 #endif
 
-        private void CheckBacklogForTimeouts() // check the head of the backlog queue, consuming anything that looks dead
+        /// <summary>
+        /// Crawls from the head of the backlog queue, consuming anything that should have timed out
+        /// and pruning it accoordingly (these messages will get timeout exceptions).
+        /// </summary>
+        private void CheckBacklogsForTimeouts()
         {
```

Comment:
name nit: `CheckBacklogForTimeouts` ('backlog' singular), if we're going to have one backlog?